<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intkeypad</class>
 <widget class="QDialog" name="intkeypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="But8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But0">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButEnter">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/image/enterbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(3, 96, 126);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButTwo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButBack">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 17pt &quot;Ubuntu&quot;;
image: url(:/image/enterbutton.png);
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButOne">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/textbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButEnter_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/image/enterbutton.png);
font: 75 17pt &quot;Ubuntu&quot;;
font:bold;
border:none;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>ButTwo</zorder>
  <zorder>But6</zorder>
  <zorder>But4</zorder>
  <zorder>But5</zorder>
  <zorder>But7</zorder>
  <zorder>But9</zorder>
  <zorder>ButBack</zorder>
  <zorder>But3</zorder>
  <zorder>ButOne</zorder>
  <zorder>ButEnter</zorder>
  <zorder>But0</zorder>
  <zorder>But8</zorder>
  <zorder>ButEnter_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
